app/shared/db/project.server.ts

```diff
@@ -75,7 +75,16 @@ export const create = async ({
   const tree = await db.tree.create(db.tree.createRootInstance(breakpoints));
   const project = await prisma.project.create({
     data: {
-      userId,
+      User: {
+        connectOrCreate: {
+          create: {
+            id: userId as string,
```

Comment:
seems like a wrong thing to do, you are basically saying it can't be null, where the model says it can be null
Why can it be null on the model? If there is a valid reason - then we should bail out before

Reply:
I removed it from the model, it now cannot be null on the model